internal/logdb/kv/pebble/kv_pebble.go

```diff
@@ -374,5 +374,5 @@ func (r *KV) FullCompaction() error {
 		fk[i] = 0
 		lk[i] = 0xFF
 	}
-	return r.db.Compact(fk, lk)
```

Comment:
The last (new) argument is if compaction should run in parallel, AFAIK the old behaviour was to not parallelise so I picked false here.